docs/changelog.txt

```diff
@@ -113,6 +113,7 @@ Template for new versions:
 - ``Units``: new ``isWildlife`` and ``isAgitated`` property checks
 - ``Items::createItem``: removed ``growth_print`` parameter; now determined automatically
 - ``DFHack::cuboid``: ``cuboid::clampMap`` now returns the cuboid itself (instead of boolean) to allow method chaining; call ``cuboid::isValid`` to determine success
+- ``DFHack::Units``: new function ``setPathGoal``
```

Comment:
should be moved up to the "Future" section above. Also, the Lua variant should be mentioned in the Lua section